modin/core/storage_formats/pandas/parsers.py

```diff
@@ -292,7 +293,9 @@ def single_worker_read(cls, fname, **kwargs):
 
 @doc(_doc_pandas_parser_class, data_type="CSV files")
 class PandasCSVParser(PandasParser):
+    # Disable logging for a function that is executed in a remote context, see TextFileDispatcher._launch_tasks
     @staticmethod
+    @disable_logging
```

Comment:
this particular piece feels weird, could you please provide some more verbose reasoning behind it?

Reply:
Added more context for this.

Reply:
I wonder what happens in non-remote context...

Reply:
In a non-remote context (running a script that does call `PandasCSVParser.parse` without the remote logging enabled), the parse is not logged since it's invoked in a process (Ray process) that does not have access to the Modin logger.